docs/guides/08-introduction-to-image-files.md

```diff
@@ -0,0 +1,75 @@
+## Introduction to Image Markers
+
+CARTO VL supports the use of images to render point data. Image markers can be accessed directly from CARTO VL's built-in symbol library, or from external sources via an HTTP request. Any image (built-in or external) can be colored, sized and placed using CARTO VL expressions.
+
+### Built-in images
+
+To access image markers from the built-in symbol library, use the `symbol` property with the name of the image:
+
+```js
+symbol: cross
+```
+
+http://127.0.0.1:8080/examples/editor/index.html#eyJhIjoidGFibGVfMzExX2NvbXBsYWludHMiLCJiIjoiIiwiYyI6ImNhcnRvdmwiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6InN5bWJvbDogY3Jvc3NcbndpZHRoOiAyMCIsImYiOnsibG5nIjotNzMuOTU4MDkyMjE5NjM1NjYsImxhdCI6NDAuNzA3ODUyMzU5MTg3MjN9LCJnIjoxNS40ODAwMzA5NDk0MjMzNDgsImgiOiJWb3lhZ2VyIn0=
+
+### External images
+
+External image markers can be loaded with a URL link. 
+
+Note: Keep in mind that the server must respond with the appropriate CORS headers for the image file to be properly loaded.
+
+To access external images, use the `symbol` property with the image URL inside of the `sprite` expression:
+
+```js
+symbol: image('https://libs.cartocdn.com/carto-vl/assets/marker.svg')
```

Comment:
Since we are changing the name to `image` this PR should contain the respective code change (https://github.com/CartoDB/carto-vl/issues/668). Let me take care of that.